scripts/pre_commit_linter.py

```diff
@@ -1357,6 +1368,59 @@ def _check_html_indent(all_files, debug=False):
     return summary_messages
 
 
+def _check_for_copyright_notice(all_files):
+    """This function checks whether copyright notice is
+    present at the beginning of files.
+    """
+
+    js_files_to_check = [
+        filename for filename in all_files if filename.endswith('.js') and (
+            not filename.endswith(GENERATED_FILE_PATHS)) and (
+                not filename.endswith(CONFIG_FILE_PATHS))]
+    py_files_to_check = [
+        filename for filename in all_files if filename.endswith('.py') and (
+            not filename.endswith('__init__.py'))]
+    sh_files_to_check = [
+        filename for filename in all_files if filename.endswith('.sh')]
+    all_files_to_check = (
+        js_files_to_check + py_files_to_check + sh_files_to_check)
+    regexp_to_check = r"Copyright \d+ The Oppia Authors\. All Rights Reserved\."
```

Comment:
`r'Copyright \d{4} The Oppia Authors\. All Rights Reserved\.'`

Reply:
This is so that the date is in the proper format.

Reply:
Done, thanks!